serverless/unsubscribe-digest/src/utils/cronitor.ts

```diff
@@ -0,0 +1,54 @@
+import axios from 'axios'
+import { Cronitor } from '../interface'
+import config from '../config'
+import { Logger } from './logger'
+
+const logger = new Logger('db-backup')
+
+const CRONITOR_PING_URL = `https://cronitor.link`
+const CRONITOR_CODE = config.get('cronitor.code')
+
+const buildUrl = (action: string, message?: string): string => {
+  const url = `${CRONITOR_PING_URL}/${CRONITOR_CODE}/${action}`
+  return message ? `${url}?msg=${message}` : url
```

Comment:
if you're really going to do this, best use the url package? otherwise you have to sanitize/url encode `message`.